### PR TITLE
Scavengers: make the queue broadcastable

### PR DESCRIPTION
**Pull request by sparkychildcharlie**
I hope this way of dealing with broadcasting in a command called via this.parse() is fine.

### CODE REVIEW

File: chat-plugins/scavengers.js [hunk 1]
```diff
@@ -1006,7 +1006,7 @@ let commands = {
 	 */
 	queue: function (target, room, user) {
 		if (room.id !== 'scavengers') return this.errorReply("This command can only be used in the scavengers room.");
-		if (!target) return this.parse("/scavengers viewqueue");
+		if (!target) return this.parse(this.message.replace('queue', 'viewqueue'));
```
Comment (whalemer) [inline chat-plugins/scavengers.js]: i think the standard for this is ``this.parse(`${this.cmdToken}scavengers viewqueue`)``, see https://github.com/Zarel/Pokemon-Showdown/blob/master/chat-commands.js#L170

**Comment by xfix** [comment]
If I type ``!scav queue``, I get ``!scav viewqueue`` back, is it intended?

**Comment by sparkychildcharlie** [comment]
OK simply directly calling the command function while binding the same command context should work.  Since there is no recursion in the viewqueue command, recursion depth should not be an issue.  This makes so that no matter how they call the broadcast the command (viewqueue/queue, or !scavs vs !scavengers) it'll show the actual message they sent.

**Comment by sparkychildcharlie** [comment]
dot dot dot bump?

**Comment by xfix** [comment]
Pushed to master, sorry for making a bad commit in your branch, I don't understand GitHub merge failure resolution tools.

**Pull request closed without merging**